ApiSurface.DocumentationSample/KitchenSink.fs

```diff
@@ -200,23 +200,23 @@ type internal MomentActionsId =
 type IndexedProperties<'a, 'b> =
     /// Simple getter with one argument
     member this.Item
-        with get (_ : int) : string = failwith ""
```

Comment:
This one is interesting actually.
As `fsharp_space_before_colon=true` is listed in the settings we no longer seem to respect this.
This is most likely a regression after Dallas.
Could you try and find out what happened here?

Reply:
Sure, I'll have a look.